storage_v2/spanstore/reader.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package spanstore
+
+import (
+	"context"
+	"time"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+// Reader finds and loads traces and other data from storage.
+type Reader interface {
+	// GetTrace retrieves the trace with a given id.
+	//
+	// If no spans are stored for this trace, it returns ErrTraceNotFound.
```

Comment:
`ErrTraceNotFound` - should be defined in the same package?

Reply:
Rather than duplicating the one from v1 storage, you can define this one as an alias, e.g.
```
var ErrTraceNotFound = storage_v1.spanstore.ErrTraceNotFound
```

Reply:
Yes, added it